app/src/main/java/com/cortado/minionify/accumulators/PreferenceAccumulator.java

```diff
@@ -0,0 +1,33 @@
+package com.cortado.minionify.accumulators;
+
+import android.content.SharedPreferences;
+
+import com.cortado.minionify.mode.AppMode;
+
+public class PreferenceAccumulator implements IAccumulator {
+    private SharedPreferences mPreferences;
+
+    private static final String KEY_NOT_FOUND_DEFAULT_VALUE = " ";
```

Comment:
choose something weirder like "!@55" or smth like that

Reply:
Done.